src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -1241,17 +1250,14 @@ public static void AnalyzeTargetHelper(TContext context, IEnumerable<Skimmer<TCo
                         ? Stopwatch.StartNew()
                         : null;
 
-                    DriverEventSource.Log.RuleStart(context.CurrentTarget.Uri.GetFilePath(), skimmer.Id, skimmer.Name);
+                    DriverEventSource.Log.RuleStart(filePath, skimmer.Id, skimmer.Name);
                     skimmer.Analyze(context);
-                    DriverEventSource.Log.RuleStop(context.CurrentTarget.Uri.GetFilePath(), skimmer.Id, skimmer.Name);
-
-                    Uri uri = context.CurrentTarget.Uri;
+                    DriverEventSource.Log.RuleStop(filePath, skimmer.Id, skimmer.Name);
 
                     if (stopwatch != null)
                     {
-                        string file = uri.GetFilePath();
-                        string directory = Path.GetDirectoryName(file);
-                        file = Path.GetFileName(file);
+                        string directory = Path.GetDirectoryName(filePath);
+                        string file = Path.GetFileName(filePath);
 
```

Comment:
There is no functionality change here, just re-use the variable added at the beginning of the function.